src/EditorFeatures/CSharpTest/RawStringLiteral/RawStringLiteralCommandHandlerTests.cs

```diff
@@ -120,6 +118,105 @@ public void TestReturnInSixQuotes_Interpolated()
     """"""");
         }
 
+        [WpfFact]
+        public void TestReturnInSixQuotesMoreQuotesLaterOn()
```

Comment:
Not sure whether this is meant to be in this PR, or an artifact of targeting 17.3 branch.